cli.js

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env node
+console.log("Hello Hydra")
```

Comment:
The "Hail Hydra" was from the marvel comics and tv shows (especially Marvels Agents of S.H.I.E.L.D.) https://en.wikipedia.org/wiki/Hydra_(comics)

Do you think its too controversial?

Reply:
I must be honest I thought this came from more German real history, I originally wrote it and then removed but I am happy to write it I like it. since you are suggesting. it I will add it.